### ISSUE TITLE
unpublish @mineral-ui component packages

### ISSUE THREAD

**Issue by coldpour** [mineral-ui/mineral-ui#349]
dependent on #348 

This should help improve our search results as it will hopefully no longer be populated with outdated npm package listings.

**Comment by brentertz** [comment]
I have sent a request to npm to delete/unpublish all of the @mineral packages, with the exception of react-popper as I wasn't sure yet if it was in use and I wanted to check with @coldpour first.

I will update/close this issue once I hear back from npm.

**Comment by brentertz** [comment]
All of the @mineral-ui scoped packages, with the exception of `react-popper` have been deleted.  This is lower priority and can be done at a later time.  

https://www.npmjs.com/org/mineral-ui
